easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.9-foss-2019a-R-3.6.0.eb

```diff
@@ -101,10 +101,8 @@ exts_list = [
         'checksums': ['d72cd4a1ba3fb14629aad3685202388384673ef06d2d22451c4c3d174a85e8e8'],
     }),
     ('preprocessCore', '1.46.0', {
-        'checksums': ['a062860681595930b98c98403d44242284a0692da197b7d42eac6e0a05b13e46'],
-        # disable internal threading to avoid crash in pthread_create
-        # see also https://github.com/bmbolstad/preprocessCore/issues/1
```

Comment:
@smoors Please restore this comment (that's bug in `--inject-checksums` we have to fix, see also https://github.com/easybuilders/easybuild-framework/issues/3082)